pkg/webhook/admission/http.go

```diff
@@ -136,11 +136,11 @@ func (wh *Webhook) writeAdmissionResponse(w io.Writer, ar v1.AdmissionReview) {
 		}
 	} else {
 		res := ar.Response
-		if log := wh.getLogger(nil); log.V(1).Enabled() {
+		if log := wh.getLogger(nil); log.V(4).Enabled() {
```

Comment:
It is technically a breaking change in that the logs may disappear when running on v1, right?

Reply:
Changed the PR title

Reply:
I don't have full context and I defer the decision to the team, but as far as I remember in previous versions of controller runtime those logs lines were not generated, so it could also be considered a fix...

Reply:
Oh okay, is his new relative to 0.14?

Reply:
Just checked. The behavior was the same with 0.14. We just didn't notice it before as we are running our controllers per default with log-level 0.

The recommended production log level is 2 though IIRC. I couldn't think a quote for that.